tracer/src/Datadog.Trace/Util/System.Runtime.CompilerServices.Attributes.cs

```diff
@@ -15,6 +15,11 @@
 namespace System.Runtime.CompilerServices;
 
 #if !NET5_0_OR_GREATER
+
+internal static class IsExternalInit
+{
+}
```

Comment:
We resisted for so long... _Finally_ we can have records! I assume our fears were unfounded, yes?

Reply:
_ominous music_